src/main/java/mate/academy/service/OrderServiceImpl.java

```diff
@@ -25,6 +30,7 @@ private List<Product> getAllProductsFromShoppingCart(Long userId) {
         Product xiaomi = new Product("Xiaomi 12", BigDecimal.valueOf(499));
         List<Product> products = List.of(iphone, macBook, xiaomi);
         // TODO: add log message about successful fetched data from DB
+        logger.info("Data from DB was successful fetched for user with ID: " + userId);
```

Comment:
Similarly, the log message in the 'getAllProductsFromShoppingCart' method should use parameterized logging for better performance and readability.